LayoutTestBase/UIViewHelpers/UIView+LYTFrameComparison.h

```diff
@@ -37,6 +48,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (BOOL)lyt_after:(UIView *)otherView NS_SWIFT_NAME(after(_:));
 
+/**
+ Version of lyt_after that takes a fromCenter param. If fromCenter is YES, the comparison starts from the center of the view rather than the
+ edge, allowing an overlap of up to half of the view. So for example, if this view is at x=6 with a width=10, and another view is at x=0, width=10,
+ the function will return YES if fromCenter==YES, NO otherwise. The function honors the epsilon vlaue and right-to-left.
+ 
+ \param otherView The view you want to compare to. It does not need to have the same superview, but MUST share some ancestor view.
+ \param fromCenter If YES, allow an overlap up to the center of otherView. If NO, behavior is the same as without the param
+ \returns True if the view is laid out before another view, allowing for an overlap up to the center if fromCenter is YES.
+ */
+- (BOOL)lyt_after:(UIView *)otherView : (BOOL)fromCenter NS_SWIFT_NAME(after(_:_:));
+
```

Comment:
All of the comments from above, apply here as well.

Reply:
Fixed